helm/cilium/templates/hubble-ui/_nginx.tpl

```diff
@@ -46,7 +46,7 @@ server {
             {{- if not (eq .Values.hubble.ui.baseUrl "/") }}
             rewrite ^{{ (trimSuffix "/" .Values.hubble.ui.baseUrl) }}(/.*)$ $1 break;
             {{- end }}
-            # double `/index.html` is required here
+            # double `/index.html` is required here 
```

Comment:
Space at the end of the line that was fixed in our chart templates. It's fine adding the space back and keeping it in sync with upstream.